### ISSUE TITLE
1k Respect Issue

### ISSUE THREAD

**Issue by Rustymaple** [Crito-VanaheimServers/XM8-Air-Strike.#1]
Love the idea of adding this to my server, but I cannot get it to work. Always get an error message saying I need at least 1k respect even though I have well over that amount.

**Comment by Crito-VanaheimServers** [comment]
in the AirStrike.sqf find these lines

_playerRespect = player getVariable ["ExileScore", 0];
if (_playerRespect < 1000) exitWith
	{
		["ErrorTitleAndText", ["Air strike", "You need at least 1000 respect to call an air strike"]] call ExileClient_gui_toaster_addTemplateToast;
		uiNamespace setVariable ["tag_cooldown",(diag_tickTime + 0)];
	};

and remove them this will not check for the respect amount anymore. I did notice on my server if I first joined and tried to use it than I got this message as well but once I died and respawned all was good after that.

**Comment by Rustymaple** [comment]
Thanks for the quick response.

I tried this before I posted here, but now it seems to do nothing at all. The error message went away, but nothing happens after clicking the button. It doesn't deduct any respect, no jets spawn. The cooldown timer does work so I know it is at least acknowledging that I've called for an air strike.

**Comment by Crito-VanaheimServers** [comment]
Do you have the server pbo in the @ExileServer/addons folder? this handles the respect server side.

**Comment by Rustymaple** [comment]
Yep. I'm currently reloading my back and I'm going to reload all the air strike stuff from scratch to see if I botched somewhere else.

**Comment by Crito-VanaheimServers** [comment]
this is my discord link https://discord.gg/HMFVyAy you can join it and @Crito in the channel or DM me and maybe we can do a scree share and help you out better.
